src/instrument/core/run_engine_init.py

```diff
@@ -31,13 +32,20 @@
 
 # Save/restore RE.md dictionary, in this precise order.
 if MD_PATH is not None:
+    handler_name = re_config.get("MD_STORAGE_HANDLER", "PersistentDict")
     try:
-        RE.md = bluesky.utils.PersistentDict(MD_PATH)
-    except Exception as e:
+        handler = {
+            "PersistentDict": bluesky.utils.PersistentDict,
+            "StoredDict": StoredDict,
```

Comment:
Seems like we are instantiating the StoredDict class exclusively in this dictionary, however at no point are we passing arguments or using the class to do anything. Am I missing something

Reply:
Here, the StoredDict replaces the default `RE.md` structure: https://github.com/BCDA-APS/bs_model_instrument/blob/54a4d2c8b22f3572470559b184e0003be69cbff1/src/instrument/core/run_engine_init.py#L41

Reply:
This commit 96657d2774 makes this more obvious.